platform_versioned_docs/version-23.3.0/enterprise/kubernetes.mdx

```diff
@@ -68,82 +71,82 @@ Seqera Platform Enterprise is distributed as a collection of Docker containers a
     ```
 
 :::note
-The `configmap.yml` manifest includes both the `tower.env` file and `tower.yml` file. These files are made available to the other containers through volume mounts.
+The `configmap.yml` manifest includes both the `tower.env` and `tower.yml` files. These files are made available to the other containers through volume mounts.
 :::
 
 ### Redis
 
 Seqera Enterprise requires a Redis database for caching purposes. Configure Redis manually by deploying a manifest to your cluster, or configure a managed Redis service.
 
-**Deploy a Redis manifest to your cluster**
+#### Deploy a Redis manifest to your cluster
```

Comment:
The list that follows the three below has them in a different order (Amazon, Azure, Google). :)